src/parser/jobs/war/About.js

```diff
@@ -0,0 +1,21 @@
+import React, {Fragment} from 'react'
+import {Icon, Message} from 'semantic-ui-react'
+
+import CONTRIBUTORS from 'data/CONTRIBUTORS'
+import CoreAbout from 'parser/core/modules/About'
+
+export default class About extends CoreAbout {
+	description = <Fragment>
```

Comment:
Can I just get an update on the `description` to reflect the current state of the module group? Something something Please Look Forward To It something.